packages/default/scss/appbar/_layout.scss

```diff
@@ -0,0 +1,64 @@
+@include exports("appbar/layout") {
+
+    // Appbar
+    .k-appbar {
+        @include use-background-clip();
+        margin: $appbar-margin-y $appbar-margin-x;
+        padding: $appbar-padding-y $appbar-padding-x;
+        border-width: $appbar-border-width;
+        border-style: solid;
+        box-sizing: border-box;
+        outline: 0;
+        font-family: $appbar-font-family;
+        font-size: $appbar-font-size;
+        line-height: $appbar-line-height;
+        list-style: none;
+        display: flex;
+        flex: 0 0 100%;
+        flex-direction: row;
+        flex-wrap: wrap;
+        align-items: center;
+        overflow: hidden;
+        -webkit-touch-callout: none;
+        -webkit-tap-highlight-color: $rgba-transparent;
+
```

Comment:
you are missing a generic
```scss
.k-appbar > * {
    flex-shrink: 0;
}
```

to target any content elements.

Also, I am not seeing spacing here. Hint: since this is a new component and gap for is finally supported in chromium, you can use it.